app/models/stocks/price.rb

```diff
@@ -0,0 +1,35 @@
+require "json"
+require "redis"
+
+class Stocks::Price
+  REDIS_HOST = "redis://redis:6379/"
```

Comment:
We should use the configure this in the application config,  enabling the redis cache, and use that.